.changeset/moody-eggs-smash.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core-plugin-api': patch
+'@backstage/core-app-api': patch
+---
+
+Allow defining application level feature flags
```

Comment:
Maybe we can expand upon this with a short example on how to? Or you can link them to the `README` file with the example 😄